qt/scientific_interfaces/ISISReflectometry/GUI/Runs/RunsView.h

```diff
@@ -113,9 +112,8 @@ class MANTIDQT_ISISREFLECTOMETRY_DLL RunsView
   RunsViewSubscriber *m_notifyee;
 
   boost::shared_ptr<SearchModel> m_searchModel;
-  std::vector<std::unique_ptr<DataProcessor::QtCommandAdapter>> m_commands;
 
-  // the interface (uses actions owned by m_commands)
+  // the interface
```

Comment:
Can these comments be removed? They tell me nothing I can't already work out from the variable names and types.